boot/freeldr/freeldr/arch/i386/pcmem.c

```diff
@@ -494,11 +495,11 @@ ReserveMemory(
     for (i = 0; i < PcMapCount; i++)
```

Comment:
PcMapCount doesn't seem to be correlated with MemoryMap variable anymore... See my comment down.